app/Main.hs

```diff
@@ -39,8 +40,10 @@ run options = withoutCursor $ do
     terms <- pure . lines =<< getContents
     renderHeader terms
 
+    languageConfigurations <- return loadConfigurations
```

Comment:
I did `return loadConfigurations` to fake it out so it looks like IO (since it will be when reading from a file).